docs/secure_manager.md

```diff
@@ -310,23 +310,23 @@ The **[Transport Layer Security (TLS)](https://en.wikipedia.org/wiki/Transport_L
 Before starting work the _Home Edge Network_, need to create a key infrastructure (for a certification authority and for each node separately).
 
 There are two scripts: 
- * [tools/gen_ca_cert.sh](../tools/gen_ca_cert.sh) for generation of CA Root Certificate, that consists of a private key `ca.key` and a self-signed root certificate `ca.crt`
- * [tools/gen_hen_cert.sh](../tools/gen_hen_cert.sh) for generation of Home Edge Node (HEN) Certificate, that consists of a private key `hen.key` and a signature HEN certificate: `hen.crt`
+ * [tools/gen_ca_cert.sh](../tools/gen_ca_cert.sh) for generation of CA Root Certificate, that consists of a private key `ca-key.pem` and a self-signed root certificate `ca-crt.pem`
+ * [tools/gen_hen_cert.sh](../tools/gen_hen_cert.sh) for generation of Home Edge Node (HEN) Certificate, that consists of a private key `hen-key.pem` and a signature HEN certificate: `hen-crt.pem`
  
 #### CA Root Certificate
 Generating a CA Root Certificate only needs to be done once for _Home Edge Network_ by running command:
 ```shell
 tools/gen_ca_cert.sh
 ```
-As a result, we get the private key `cert/ca.key` and self-signed root certificate `cert/ca.crt`. The certificate must be saved on all nodes in the `/var/edge-orchestration/data/cert/` folder.
+As a result, we get the private key `certs/ca-key.pem` and self-signed root certificate `certs/ca-crt.pem`. The certificate must be saved on all nodes in the `/var/edge-orchestration/certs/` folder.
 
 #### Home Edge Node (HEN) Certificate
-Generate HEN Certificate private key : hen.key
+Generate HEN Certificate private key : hen-key.pem
 
-For each node, need to create a private key `hen.key` and a certificate `hen.crt`. To do this, need to start the script and specify the node IP address. 
+For each node, need to create a private key `hen-key.pem` and a certificate `hen-crt.pem`. To do this, need to start the script and specify the node IP address. 
 ```shell
 tools/gen_hen_cert.sh 192.168.0.100
 ```
-As a result, the key and certificate will be created in the `cert/<IP>/`. They must be copied into the `/var/edge-orchestration/data/cert/` folder on a node with the specified IP address
+As a result, the key and certificate will be created in the `certs/<IP>/`. They must be copied into the `/var/edge-orchestration/certs/` folder on a node with the specified IP address
```

Comment:
Can we do this without `IP`? MQTT? MQTT broker stores each certificate files in the `/etc/mosquitto` folder and I don't think several devices can connect to the broker with their own certificate files.

Reply:
'/etc/mosquitto/certs ' folder contains the `ca-crt.pem` certificate, MQTT broker `hen-crt.pem` certificate and `hen-key.pem` private key. This is required for two-way authentication between MQTT broker and client. The MQTT broker has its own IP address and a key and a certificate are generated for it in the same way as for any Home Edge Node (hen).

Reply:
Sorry I do not know much. It means that any home edge node and broker have different `hen-crt.pem` and `hen-key.pem` but same `ca-crt.pem`?

Reply:
Right!